spec/std/isa/inst/Zcmt/cm.jalt.yaml

```
@@ -0,0 +1,64 @@
+# Copyright (c) Ventana Micro Systems
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../../../schemas/inst_schema.json
+
+$schema: "inst_schema.json#"
+kind: instruction
+name: cm.jalt
+long_name: Jump Via Table with Optional Link
+description: |
+  Read an address from the Jump Vector Table and jump to it, linking to `ra`.
+definedBy: Zcmt
+assembly: index
+encoding:
+  match: 101000--------10
+  variables:
+    - name: index
+      location: 9-2
+      # prettier-ignore
+      not: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31 ]
+access:
+  s: always
+  u: always
+  vs: always
+  vu: always
+operation(): |
+  # Ensure JVT readable
+  Csr csr_handle = direct_csr_lookup(CSR[jvt].address());
+  XReg jvt = csr_sw_read(csr_handle);
```

actually, since what you want out of JVT isn't affected by JVT's sw_read(), you can just access JVT directory

```suggestion
  XReg jvt = $bits(CSR[jvt]);
```

---

What I was attempting to achieve with reading `jvt` via its `sw_read()` method was to respect this from the spec:
> If the Smstateen extension is implemented, then bit 2 in mstateen0, sstateen0, and hstateen0 is implemented. If bit 2 of a controlling stateen0 CSR is zero, then access to the jvt CSR and execution of a cm.jalt or cm.jt instruction by a lower privilege level results in an illegal-instruction trap (or, if appropriate, a virtual-instruction trap).

All of this is already encoded in that method, but would be unused with a direct access.